docs/auth/identity-resolver.md

```diff
@@ -96,7 +96,7 @@ It can be enabled like this
 
 ```tsx
 // File: packages/backend/src/plugins/auth.ts
-import { googleEmailSignInResolver } from '@backstage/plugin-auth-backend';
+import { googleEmailSignInResolver, createGoogleProvider, createRouter } from '@backstage/plugin-auth-backend';
```

Comment:
Nitpick: you can skip the `createRouter` probably, since this is a diff it just wanted to show what to add and that one was there from before :)

Reply:
done